package.json

```
@@ -22,7 +22,7 @@
     "etag": "^1.8.1",
     "fresh": "^2.0.0",
     "http-errors": "^2.0.0",
-    "mime-types": "^2.1.35",
+    "mime-types": "^3.0.0",
```

You're still going to release mime-types as a patch.
```suggestion
    "mime-types": "^3.0.1",
```